Makefile

```diff
@@ -113,6 +113,69 @@ distclean:
 
 check: test
 
+# Remove files generated by running coverage, put the un-intstrumented lib back 
```

Comment:
Hmmm... or even instrumented ?  I guess this is what comes from years of muscle memory used to typing `int` repeatedly